.github/workflows/ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           java-version: 11
           architecture: ${{ matrix.architecture }}
       - name: Build with Gradle
-        run: ./gradlew build
```

Comment:
Is this change intentional?

Reply:
Yes. This will enable the arm64 builds. The flag is ignored everywhere except the x64 windows build. So no need to if out this.